uploader/api_commands.py

```diff
@@ -31,15 +28,19 @@ def upload_video():
     extension = os.path.splitext(filename)[1]
     filename_with_timestamp = f"{filename_without_ext}_{current_time}{extension}"
 
-    # TODO: We need to use the DB container to store the file url with status 'unprocessed'
-    file.save(os.path.join(current_unprocessed_folder, filename_with_timestamp))
+    
+    # TODO: We need to use the DB container to store the file url with status 'uploaded'
+    video = Video(status=Status.uploaded,uploaded_file_url=os.path.join(current_unprocessed_folder, filename_with_timestamp),created_on=datetime.datetime.now())
+    db.session.add(video)
+    db.session.commit()
 
-    # TODO: We need to return here the file id to the user
+    file.save(os.path.join(current_unprocessed_folder, filename_with_timestamp))
```

Comment:
Que pasa si el `file.save` falla ? 
se creara un registro en la base de datos, con informacion falsa.
podemos invertir el flujo ?
```suggestion
file.save(os.path.join(current_unprocessed_folder, filename_with_timestamp))

video = Video(status=Status.uploaded,uploaded_file_url=os.path.join(current_unprocessed_folder, filename_with_timestamp),created_on=datetime.datetime.now())
    db.session.add(video)
    db.session.commit()
```

Reply:
Listo.  Igual no deberiamos enredarnos con edge cases...